.changes/unreleased/tinymce-TINY-10885-2024-04-30.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Edit image modal did lose focus after image upload.
```

"edit image" is a feature now, so we probably need to be more specific
```suggestion
body: Insert/Edit image dialog lost focus after the image upload completed.
```